src/views/AccountDetail.vue

```diff
@@ -48,7 +48,7 @@
         </template>
       </Card>
 
-      <Card v-if="1||showTransactions" class="card-f card-full-width" :loading="loading">
```

Comment:
i think `showTransactions ` is valild because it existing in `computed` , only `1||` have to remove.